static/css/app.css

```diff
@@ -27,61 +198,272 @@ input {
 }
 
 a {
-  color: #25172a;
+  color: var(--link-color, #25172a);
+}
+
+a[href="#"] {
+  color: var(--muted-color, #757575);
+}
+
+ul {
+  padding: 0;
+}
+
+li {
+  list-style: none;
+  margin-bottom: 0.4rem;
+}
+
+li > * {
+  vertical-align: middle;
 }
 
 input {
   font-family: "Roboto Mono", "Monaco", monospace;
-  background-color: #eee;
-  border: 1px solid #ccc;
-  color: black;
+  background-color: var(--input-background-color, #eee);
+  border: 1px solid var(--border-color);
+  color: var(--text-color, #000);
   width: 25em;
+  padding: 0.4rem 0.5rem;
+  max-width: 100%;x
 }
 
 blockquote {
-  background-color: #ddd;
+  background-color: var(--quote-background, #ddd);
 }
 
 blockquote blockquote {
   margin: 0;
-  background-color: #aaa2;
+  background-color: var(--quote-2-background, #aaa4);
 }
 
 pre {
-  background-color: #d7d9db;
+  background-color: var(--code-background, #d7d9db);
   margin: 0;
+  padding: 0.5rem;
+}
+
+label {
+  margin-block-end: 0.8rem;
+  display: block;
+}
+
+label > img {
+  margin-inline-end: 0.8rem;
+}
+
+code {
+  white-space: pre-wrap;
+}
+/*** COMPOSITION ***/
+.container {
+  width: 100%;
+  max-width: 63.75rem;
+  margin: auto;
+  padding: 1.25rem;
+}
+
+.container p > img:only-child {
+  max-width: 100%;
+}
+
+.inline-list {
+  padding: 1rem;
+  display: flex;
+  justify-content: space-between;
+}
+
+
+
+.stack,
+.stack > p {
+    display: flex;
+    flex-direction: column;
+}
+
+.stack > p {
+  margin: 0;
+}
+
+.stack > a,
+.stack > p > a{
+    margin-block-end: 0.4rem;
+}
+
+.columns-2,
+.columns-3 {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  grid-gap: 3.75rem;
+  margin: 3.75rem auto;
+}
+
+.footer {
+  margin-block-start: 2rem;
+  background-color: var(--header-background, #d7d9db);
+  border-top: 1px solid var(--border-color);
+}
+.footer > img {
+  display: block;
+  margin: 1rem auto;
+}
+
+/** 51.2rem **/
+@media screen and (min-width: 68.75rem) {
+  .stack,
+  .stack > p {
+    flex-direction:  row;
+  }
+  .stack *:not(:first-child) {
+    margin-left: 3.75rem;
+  }
+  .stack > a,
+  .stack > p > a{
+    margin-block-end: 0;
+  }
+  .columns-2 {
+    grid-template-columns: repeat(2, 1fr);
+  }
+  .columns-3 {
+    grid-template-columns: repeat(3, 1fr);
+  }
+}
+
+/*** UTILITIES ***/
+
+.is-hidden {
+  display: none;
+}
+
+.is-muted {
+  color: var(--muted-color, #757575);
+}
+
+.is-finished {
+  text-decoration: line-through;
+}
+
+.is-underline {
+  text-decoration: underline;
+}
+
+/*** BLOCKS ***/
+.tabs button {
+  border: none;
+  cursor: pointer;
+  text-decoration: underline;
+  padding: 0;
+  background: none;
+  color: var(--text-color, #000);
+}
+
+.tabs button[aria-selected="true"] {
+  font-weight: 700;
+}
+
+.tabs + .jumbotron {
+  margin-top: 2.5rem;
+}
+.tabs > .columns-2,
+.tabs > .columns-3 {
+  margin-bottom: 2.5rem;
+}
+
+.accordion-trigger {
```

Comment:
approving all these for now;

would be good to have as goal the creation of a framework for user gno.land generated markdown to benefit from standard *minimal* components, so please help curate the classes.

Reply:
@jaekwon Yes, it's exactly what these classes are built for. I have already started generating this UI lib in this [PR](https://github.com/gnolang/gno/pull/781). Nevertheless, due to our website release deadline, I had to rollover, re-prioritize the website over the lib and use the classes only (before using the UI lib soon asap).

Reply:
yep, it actually started with smart contracts, that's a joint effort between alexis and I to make a UI framework for smart contract developers.

Also linked with:
* https://github.com/gnolang/gno/issues/903
* https://github.com/gnolang/gno/issues/439